Makefile

```
@@ -0,0 +1,17 @@
+mutation:
```

Alguna vez hizo una encuesta Evaristo y ganó `make mutants` tal vez lo podrías cambiar para irlo probando
```suggestion
mutants:
```